### ISSUE TITLE
500 Internal Server Error && Error in getrapache("receiveBin")() :    'Realloc' could not re-allocate memory (18446744071562067968 bytes)

### ISSUE THREAD

**Issue by chinsoon12** [opencpu/opencpu#265]
Hi,

**Problem Description**
Getting following issue when reading a large file with `data.table::fread` via opencpu using `curl.exe https://opencpu.rhel7server/ocpu/library/data.table/R/fread -F "input=@1GBfile.csv"`:
```
HTTP/1.1 500 Internal Server Error
Date: Fri, 20 Oct 2017 08:45:42 GMT
Server: Apache/2.4.6 (Red Hat Enterprise Linux) OpenSSL/1.0.2k-fips mod_R/1.2.7 R/3.4.1 mod_apreq2-20090110/2.8.0
Content-Length: 527
Connection: close
Content-Type: text/html; charset=iso-8859-1

<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head><title>500 Internal Server Error</title></head><body>
<h1>Internal Server Error</h1>
<p>The server encountered an internal error or misconfiguration and was unable to complete your request.</p>
<p>Please contact the server administrator at root@localhost to inform them of the time this error occurred, and the actions you performed just before this error.</p>
<p>More information about this error may be available in the server error log.</p>
</body></html>
```

**Looking at logs**
In /var/log/httpd/ssl_error_log
```
Error in getrapache("receiveBin")() : 
  'Realloc' could not re-allocate memory (18446744071562067968 bytes)
Traceback:
No traceback available 
```

**Trying online suggestions**
Have increased rlimit.as and rlimit.fsize in /etc/opencpu/server.conf
```
    "rlimit.as": 100e9,
    "rlimit.fsize": 16e9,
```
but getting the same error.

This number, 18446744071562067968, looks way too huge for a 1GB file. 
And I am able to read an even larger file using R on the server. 
Also checked the available disk space in /tmp using `df -hl` (has available space of around 7.8G)

**Fix?**
Might be related to [#130 Debugging info for R process death](https://github.com/opencpu/opencpu/issues/130)

Anything else I can try, fix or figure out how this issue came about? Preferably not creating databases as that might be too technical for the end user.

**Comment by jeroen** [comment]
Hmm interesting. The `18446744071562067968` looks like an integer overflow. From your error message it seems that the error appears within rapache.

Does the same issue appear for any function call, or only `fread`? Can you try posting it to e.g.
`/ocpu/library/base/R/file.info`?

**Comment by jeroen** [comment]
Actually I can reproduce this now. Seems to be an issue in rapache, it doesn't happen in the single user server. Thanks for reporting!

**Comment by jeroen** [comment]
Hi @chinsoon12 I have published rapache 1.2.9 to dockerhub and the archive: https://archive.opencpu.org/. Can you test if upgrading rapache fixes your problem?

**Comment by chinsoon12** [comment]
thanks, @jeroen! it works! thanks, @jeffreyhorner as well!
